src/main/java/data/JsonConverter.java

```diff
@@ -134,6 +134,10 @@ public boolean configExists(String vendorID) {
     return getVendorIDs().contains(vendorID);
   }
 
+  public boolean accountExists(Vendor vendor, String accountId) {
```

Comment:
I think Vincent was looking for a function like this. Can you please communicate this change to him when he is back from OOO?

Reply:
got it